QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+$QUEST_JOBSTEP_20150317_000765	Lydia quickly liberated the traveler of his conditions, and in exchange, sent a challenge request to Rumpelstiltskin. Rumpelstiltskin was well acquainted with Lydia Schaffen's reputation, and the fact that she requested a game with him excited him into accepting her request. 
+$QUEST_JOBSTEP_20150317_000766	Although Rumpelstiltskin is not well known by people today, he played often and was famous for gifting those who lost in his games in the past. But, his games have always ended with little variation. Lydia Schaffen proposed ,"I ask for a game related to archery, but, what sort of thing should we do?"
+$QUEST_JOBSTEP_20150317_000767	"Of course, a game to see who can shoot better with a bow would not be that interesting. Therefore, let's play a game where you fire an arrow with your bow at the target, and then, I will try to catch and ride on the arrow before the arrow reaches the target," said Rumpelstiltskin. Lydia replied back, questioning the proposal,"Ride on the arrow?" "That is correct," averred Rumpelstiltskin.
+$QUEST_JOBSTEP_20150317_000768	Upon answering, Rumpelstiltskin started to shrink his body. Once he had reduced himself to the size of a child's fist, Rumpelstiltskin spoke ,"Well, this size will allow me to ride on your arrow, right?" Lydia Schaffen thought for a bit ,"That's a problem for the match. If you touch the arrow and cause it to fall to the ground without it ever hitting the target, then wouldn't that still be a loss for me regardless of whether you rode the arrow or not?" Lydia answered Rumpelstiltskin with these words.
+$QUEST_JOBSTEP_20150317_000769	"Then how about this. I win if I can ride your arrow until it reaches the target," answered Rumpelstiltskin. "And if your arrow falls to the ground," he continued ,"then I will also lose." Rumpelstiltskin added ,"The game shall have 3 rounds, best 2 out of 3."
+$QUEST_JOBSTEP_20150317_000770	After Rumpelstiltskin seemingly honest answer, Lydia Schaffen replied ,"Alright. One more question though. In a place like this, what sort of targets will I shoot at? I mean you won't be used to riding the arrows until round 2 or round 3, so the targets for each round should be called in advance. Don't you agree?" "Alright, let's do that," agreed Rumpelstiltskin. And after agreeing on compensation upon defeat as well, the game between two people, err, one person and one demon began.
+$QUEST_JOBSTEP_20150317_000771	Lydia Schaffen weighed her arrow, then called out the first target ,"The tawny tree over there!" Together with those words in advance she fired her bowstring, the arrow's frightful speed sliced through the air.
```

Comment:
out the first target ,"The tawny tree over there!" -> out the first target, "The tawny tree over there!"